worker/worker_blackbox_test.go

```diff
@@ -18,23 +22,41 @@ import (
 
 type WorkerTestSuite struct {
 	gormtestsupport.DBTestSuite
+	application application.Application
+	ctx         context.Context
```

Comment:
We don't need this application and ctx anymore since you reverted the benchmark thing.

Reply:
fixed in c3c911f